<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlForm</class>
 <widget class="QWidget" name="ControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>445</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_root">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_left">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_connect">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>font-size: 14px; font-weight: bold;</string>
       </property>
       <property name="text">
        <string>CONNECT TO AGV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupManualMove">
       <property name="title">
        <string>Move</string>
       </property>
       <layout class="QGridLayout" name="gl_move">
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_up">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>▲</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btn_left">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>◀</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btn_stop">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>||</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btn_right">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>▶</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btn_down">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>▼</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hl_controls">
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="lbl_speed_text">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sld_speed">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="lbl_grab_text">
           <property name="text">
            <string>Grab</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="dial_grab">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupArmControl">
       <property name="title">
        <string>Arm (1,2,3)</string>
       </property>
       <layout class="QVBoxLayout" name="vl_arm">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_s1">
          <property name="text">
           <string>S1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sld_s1">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_s2">
          <property name="text">
           <string>S2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sld_s2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_s3">
          <property name="text">
           <string>S3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sld_s3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frameCamera">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>background-color: #000000; border: 1px solid #333; border-radius: 10px;</string>
     </property>
     <layout class="QVBoxLayout" name="vl_cam">
      <item>
       <widget class="QLabel" name="lbl_camera_view">
        <property name="styleSheet">
         <string>color: #777; font-size: 14px; border: none;</string>
        </property>
        <property name="text">
         <string>WAITING FOR CAMERA...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
